packages/core/src/log/shared.ts

```diff
@@ -137,19 +141,40 @@ export class ZWaveLogContainer extends winston.Container {
 	}
 
 	public updateConfiguration(config: DeepPartial<LogConfig>): void {
+		const changedLoggingTarget =
+			config.logToFile != undefined &&
+			config.logToFile !== this.logConfig.logToFile;
+
+		const changedLogLevel =
+			config.level != undefined && config.level !== this.logConfig.level;
```

Comment:
Is the  `!=` used on purpose instead of `!==` to catch also `null`?

Reply:
Yep, I always do that.